### PR TITLE
OpenShift field tested updates.

### PR DESCRIPTION
**Pull request by datamattsson**
Signed-off-by: Michael Mattsson <michael.mattsson@gmail.com>

### CODE REVIEW

File: docs/partners/redhat_openshift/index.md [hunk 1]
```diff
@@ -231,3 +235,17 @@ v2.storage.hpe.com
 ```
 
 Please refer to the OLM Lifecycle Manager documentation on how to safely [Uninstall your operator](https://olm.operatorframework.io/docs/tasks/uninstall-operator/).
+
+# Unsupported Helm Chart Install
+
+In the event of an OpenShift release come out in between driver releases or interest arise running the HPE CSI Driver on an uncertified version of OpenShift, it's possible to install the CSI driver with the Helm chart instead.
```
Comment (c-snell) [inline docs/partners/redhat_openshift/index.md:241]:
```suggestion
In the event Red Hat releases a new release of OpenShift between HPE CSI driver releases or if interest arises to run the HPE CSI Driver on an uncertified version of OpenShift, it's possible to install the CSI driver using the Helm chart instead.
```

File: docs/partners/redhat_openshift/index.md [hunk 2]
```diff
@@ -13,27 +13,31 @@ Software deployed on OpenShift 4 follows the [Operator pattern](https://kubernet
 
 ### Certified combinations
 
-Software delivered through the HPE and Red Hat partnership follows a rigorous certification process and only qualify what's listed in the following table.
+Software delivered through the HPE and Red Hat partnership follows a [rigorous certification process](https://redhat-connect.gitbook.io/openshift-badges/badges/container-storage-interface-csi-1) and only qualify what's listed as "Certified" in the below table.
 
 | Status                  | Red Hat OpenShift                 | HPE CSI Operator           | Container Storage Providers          |
 | ----------------------- | --------------------------------- | -------------------------- | ------------------------------------ |
+| Field Tested<sup>5</sup>| 4.12 EUS<sup>3</sup>              | 2.2.0 [using Helm](#unsupported_helm_chart_install) | [Alletra 6000](../../container_storage_provider/hpe_alletra_6000/index.md) |
```
Comment (c-snell) [inline docs/partners/redhat_openshift/index.md:20]: Why only 6K? Is there something that needs done for a 9k?

**Review by c-snell — APPROVED** [review]
A couple grammar fixes and then LGTM

File: docs/partners/redhat_openshift/index.md (hunk 2 above)
Comment (datamattsson) [reply docs/partners/redhat_openshift/index.md:20]: For certification, QA runs their testing on both platforms. Field testing should reflect what actually has been tested. I'm in the midst of getting a Primera updated to a current release and I can run through the tests suite on that once completed.

**Pull request merged**